images/git/main.tf

```diff
@@ -0,0 +1,81 @@
+terraform {
+  required_providers {
+    apko = {
+      source                = "chainguard-dev/apko"
+      configuration_aliases = [apko.alpine]
+    }
+  }
+}
+
+locals {
+  accounts = toset(["nonroot", "root"])
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "dev" { source = "../../tflib/dev-subvariant" }
+
+module "tagger" {
+  for_each = local.accounts
```

Comment:
I always worry about multiple `tagger` invocations since `tagger` exists to avoid tag collisions.  I wonder if it's worth trying to turn it into one giant map instead?  We already wait on all tests in each variant.

Reply:
I flattened it into a single module and am much happier with the outcome! I think that's about as readable as it'll get